app/models/journeys/answers_student_loans_details_updater.rb

```diff
@@ -0,0 +1,41 @@
+module Journeys
+  class AnswersStudentLoansDetailsUpdater
+    def self.call(journey_session)
+      new(journey_session).save!
+    end
+
+    def initialize(journey_session)
+      @journey_session = journey_session
+    end
+
+    def save!
+      # When the claim hasn't been submitted yet, we need a way of knowing if
+      # the student loan details on the claim were found using the SLC data we
+      # held before submission; after submission, the
+      # `submitted_using_slc_data` value must not change
+      journey_session.answers.assign_attributes(
+        has_student_loan: student_loans_data.found_data?,
+        student_loan_plan: student_loans_data.student_loan_plan,
+        submitted_using_slc_data: student_loans_data.found_data?
+      )
+
+      journey_session.save!
+    rescue => e
```

Comment:
Not stoked on a bare rescue but that's how it's done in the claim student loans details updater and I'm unsure on what errors can be raised in production.